IMAGES.rst

```diff
@@ -92,13 +98,14 @@ parameter to Breeze:
 
 .. code-block:: bash
 
-  ./breeze build-image --python 3.7 --extras=gcp --production-image --install-airflow-version=1.10.12
+  ./breeze build-image --python 3.7 --additional-extras=gcp \
+      --production-image --install-airflow-version=1.10.12
 
 This will build the image using command similar to:
 
 .. code-block:: bash
 
-    pip install apache-airflow[sendgrid]==1.10.12 \
+    pip install apache-airflow[...,gcp]==1.10.12 \
```

Comment:
I know that this is only an example, but assuming someone will copy-paste this command the `...` will not work, right? Should we use `aws` or something?